### PR TITLE
remove questionnaires page and add all questionnaires to home

### PR DESCRIPTION
**Pull request by sandrosc**

### CODE REVIEW

File: src/app/questionnaire-form-filler/questionnaire-form-filler.component.ts [hunk 1]
```diff
@@ -25,44 +25,24 @@ export class QuestionnaireFormFillerComponent implements OnInit {
...
+        id === 'radiology-order'
           ? QuestionnaireDemo.getQuestionnaireRadiologyOrder()
-          : name === '-1'
-          ? this.questionaireFillerServer.getQuestionniare()
+          : id === '-1'
+          ? this.questionnaireFillerServer.getQuestionniare()
           : undefined
       )
     );
     this.questionnaireResponseInitial$ = this.route.paramMap.pipe(
```
Comment (joelfisch) [inline src/app/questionnaire-form-filler/questionnaire-form-filler.component.ts:43]: Do we still need this?

**Review by joelfisch — APPROVED** [review]
Looks good.

File: src/app/questionnaire-form-filler/questionnaire-form-filler.component.ts (hunk 1 above)
Comment (sandrosc) [reply src/app/questionnaire-form-filler/questionnaire-form-filler.component.ts:43]: good question. right now we don't have a local questionnaire response, so we can't use this, but in the future we might add one again. what do you think?

Comment (joelfisch) [reply src/app/questionnaire-form-filler/questionnaire-form-filler.component.ts:43]: I do not think that this is the case as this was primarily to test if the pre-filling via QuestionnaireResponse would work. In the new implementation we will pre-fill with the QuestionnaireResponses from the server which is more useful (I do not see any use case where a locally saved response would be preferable).

File: src/app/questionnaire-form-filler/questionnaire-form-filler.component.ts [hunk 2]
```diff
@@ -14,9 +14,6 @@ import { FhirConfigService } from '../fhirConfig.service';
 })
 export class QuestionnaireFormFillerComponent implements OnInit {
   questionnaire$: Observable<fhir.r4.Questionnaire | undefined>;
-  questionnaireResponseInitial$: Observable<
```
Comment (joelfisch) [inline src/app/questionnaire-form-filler/questionnaire-form-filler.component.ts:17]: Do not forget to remove it from the html as well.

**Pull request merged**